### PR TITLE
feat(service-portal): add ehic card

### PR DESCRIPTION
**Pull request by thorkellmani**
## What

* Add EHIC to license service

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/api/domains/license-service/src/lib/mappers/ehicCardMapper.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          typedPayload.hasTempCard && typedPayload.tempCardPdf
+            ? {
+                label: getLabel('downloadCard', locale, label),
+                value: getDocument(typedPayload.tempCardPdf, 'pdf'),
+                type: GenericUserLicenseMetaLinksType.Download,
+              }
+            : undefined,
+          {
+            label: getLabel('applyForNewCard', locale, label),
+            value: '/umsoknir/evropska-sjukratryggingakortid',
```
Comment (thordurhhh) [inline libs/api/domains/license-service/src/lib/mappers/ehicCardMapper.ts:95]: Is this for this value?:
https://github.com/island-is/island.is/pull/12495/files#diff-531d0a7b989a8548e3a5d5d0076e87ee38c796c382fe1302b343678ea889b53cR105

**Review by thordurhhh — APPROVED** [review]
LGTM. Just one minor question 😺

File: libs/api/domains/license-service/src/lib/mappers/ehicCardMapper.ts (hunk 1 above)
Comment (thorkellmani) [reply libs/api/domains/license-service/src/lib/mappers/ehicCardMapper.ts:95]: different things, but this shouldn't be hardcoded, will fix l8r

File: libs/clients/p-card/src/clientConfig.json
Comment (saevarma) [inline libs/clients/p-card/src/clientConfig.json:1]: Looks like there is some codeowner misconfiguration as Core should probably not be a codeowner here.

**Review by saevarma — APPROVED** [review]
Core files LGTM

File: libs/clients/p-card/src/clientConfig.json
Comment (thorkellmani) [reply libs/clients/p-card/src/clientConfig.json:1]: will fix!

**Pull request merged**